prExplorer/client-app/src/app/common/form/CustomTextArea.tsx

```diff
@@ -0,0 +1,23 @@
+import { useField } from 'formik';
+import React from 'react';
+import { Form, Label } from 'semantic-ui-react';
+
+interface CustomTextAreaProps {
+    placeholder: string;
+    name: string;
+    rows: number;
```

Comment:
`rows` seems to not be used

Reply:
I use it on ProductForm.tsx 
            <CustomTextArea
              rows={3}
              placeholder="Description"
              name="description"
            />